flyteidl/protos/flyteidl/core/execution_envs.proto

```diff
@@ -36,4 +36,7 @@ message ExecutionEnv {
     // spec is a specification of the environment.
     google.protobuf.Struct spec = 4;
   }
+
+  // version is the version of the execution environment.
+  string version = 5;
```

Comment:
Worth documenting any guidance on how this field should be used (or document that it's a complete blackbox if so)?

Reply:
Updated very briefly.